openshift-ci/build-root/Dockerfile

```diff
@@ -2,7 +2,7 @@
 
 FROM registry.svc.ci.openshift.org/openshift/release:golang-1.14
 
-RUN yum -y install dnf httpd-tools sudo
+RUN yum -y install dnf httpd-tools sudo dbus-x11
```

Comment:
@bigkevmcd We can consume docker file changes in the same pr along with other changes but the Dockerfile changes won't be verified. Check the log here at - https://gcsweb-ci.apps.ci.l2s4.p1.openshiftapps.com/gcs/origin-ci-test/pr-logs/pull/redhat-developer_kam/87/pull-ci-redhat-developer-kam-master-v4.5-integration-e2e/1334147706221760512/artifacts/build-logs/root.log.

Consider a scenario -

We have a pr failing due to some executable is missing in the test [container](https://github.com/redhat-developer/kam/blob/master/openshift-ci/build-root/Dockerfile) path. 

Current prow CI approach to fix it.

1. In the Dockerfile add the required packages then commit and merge the changes into master
2. The failed PR will consume the executable added in the test container after a rebase or ```/retest``` or ```/test all```


As a solution if we do the changes in the docker file to consume the executable in the same pr, it won't work



 3: RUN yum -y install dnf httpd-tools sudo